src/modules/feature-modules/innovator/pages/innovation/overview/overview.component.ts

```diff
@@ -97,7 +99,6 @@ export class InnovationOverviewComponent extends CoreComponent implements OnInit
       this.innovationStatus = sectionSummary.innovation.status;
       this.innovationSections = sectionSummary.sections;
 
-
       this.sections.progressBar = this.innovationSections.reduce((acc: boolean[], item) => {
```

Comment:
The changes you need, should be around this part of the code. Understand what this does, and think on a alternative...